erpnext_taxjar/api.py

```diff
@@ -89,38 +99,65 @@ def create_transaction(doc, method):
 
 	try:
 		client.create_order(tax_dict)
+	except taxjar.exceptions.TaxJarResponseError as err:
+		frappe.throw(_(sanitize_error_response(err)))
 	except Exception as ex:
 		print(traceback.format_exc(ex))
 
+
 def delete_transaction(doc, method):
 	client = get_client()
 	client.delete_order(doc.name)
 
 
 def get_tax_data(doc):
-	from shipment_management.utils import get_state_code
+	company_address = get_company_address(get_default_company()).company_address
+	company_address = frappe.get_doc("Address", company_address)
+
+	if company_address:
+		if doc.shipping_address_name:
+			shipping_address = frappe.get_doc("Address", doc.shipping_address_name)
+		else:
+			shipping_address = company_address
+	else:
+		return
+
+	if not shipping_address.country == "United States":
+		return
+
+	us_states = pycountry.subdivisions.get(country_code='US')
+	us_states = [state.code.split('-')[1] for state in us_states]
```

Comment:
Remove this entirely

Reply:
ISO Codes use the Country Code as the prefix for the state code. For example, FL becomes US-FL. If I use a simple lookup with FL as the state in the form, pycountry will catch a false positive error.